src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +80,36 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
-			year, month, events, minDate, containerId, $container;
+		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ), // Ability to extend the calendar
+			year, month, events, minDate, maxDate, containerId, $container,
+			currDate = new Date(),
+			currYear = currDate.getFullYear(),
+			currMonth = currDate.getMonth();
 
 		events = getEvents( $elm );
 		containerId = $elm.data( "calevtSrc" );
 		$container = $( "#" + containerId ).addClass( componentName + "-cal" );
 
 		minDate = events.minDate;
-		year = settings.year || minDate.getFullYear();
-		month = settings.month || minDate.getMonth();
+		maxDate = events.maxDate;
+
+		if ( minDate.getTime() > currDate.getTime() ) {
```

Comment:
What @nschonni said is the value you are retrieving is from "events.minDate". And that value was set by the returned value of the local function "getEvents()" called at [line 89](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L89) in the code. Note that is the only place where the function "getEvents()" is called. So inside the function "getEvents()" starting at [line 150](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L150) there is already some calculation made in order to set the proper minDate and maxDate (between the [line 242](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L242) and [line 248](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L248)). 

So the question is why those calculated minDate and maxDate are not good? Why to modify/re-evaluate them between [line 96](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L96) and [line 102](https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L102)?